addons/common/functions/fnc_nearestVehiclesFreeSeat.sqf

```diff
@@ -22,5 +22,5 @@ private _nearVehicles = nearestObjects [_unit, ["Car", "Air", "Tank", "Ship_F",
 _nearVehicles select {
     // Filter cargo seats that will eject unconscious units (e.g. quad bike)
     ((_x emptyPositions "cargo" > 0) && {!(_unit getVariable ['ACE_isUnconscious', false])} || {(getNumber (configFile >> "CfgVehicles" >> (typeOf _x) >> "ejectDeadCargo")) == 0})
-    || {_x emptyPositions "gunner" > 0}
+    || {{isNull (_x#0)} count fullCrew [_x, "", true] > _x emptyPositions "driver"}
```

Comment:
this has same error as above (also line 24)
But after fixing it still won't be right because line 25 will ignore the ejectDeadCargo check

Reply:
I think something like this should work
```
_nearVehicles select {
    private _canSitInCargo = (!(_unit getVariable ['ACE_isUnconscious', false])) || {(getNumber (configFile >> "CfgVehicles" >> (typeOf _x) >> "ejectDeadCargo")) == 0};
    ((fullCrew [_x, "", true]) findIf {
        _x params ["_body", "_role", "_cargoIndex"];
        (isNull _body) && {_role != "DRIVER"} && {_canSitInCargo || {_cargoIndex == -1}}
    }) > -1
}
```